### ISSUE TITLE
Error when bundling

### ISSUE THREAD

**Issue by blinxess** [latte-soft/wax#2]
Whenever I run: ` lune wax bundle ` I get this error:

```
[ERROR] 
invalid argument #1 to 'match' (string expected, got userdata)
[Stack Begin]
    Script '[C]' - function match
    Script 'Wax', Line 2665 - function FormatError
    Script 'Wax', Line 2702 - function LoadScript
    Script 'Wax', Line 2921
    Script 'lune\wax', Line 21
[Stack End]
```

I don't know what the problem is, am I not supplying the correct options?

**Comment by regginator** [comment]
Thank you for filing this!

This has been an issue (on Windows exclusively) I've been aware of for a couple days, and have been working on resolving for 0.2.2 (it's quite annoying as it stems from how Lune deals with its `process.spawn` impl.

I've gotten it to the point where you can bundle **without `minify=true`** on Windows. (Soley due to Darklua not functioning with the default shell env thing for some reason)
You should now be able to clone the repo and run `lune make` to build with these fixes, but I'm trying to figure out a workaround for Darklua prior to a formal release.

Related commits:
* https://github.com/latte-soft/wax/commit/8aa18a1d7387bc8bda1c333e1e1b4d96bf155626
* https://github.com/latte-soft/wax/commit/3d4dc8989b4b561522bd6738f7db1d390f96150b

**Comment by blinxess** [comment]
I ran `lune wax bundle input=default.project.json` and now it works! Thanks alot!!!

**Comment by regginator** [comment]
> I ran `lune wax bundle input=default.project.json` and now it works! Thanks alot!!!

Additionally, I'd also note that `input=default.project.json` is actually the default now- you *could* just run `lune wax bundle` aswell!

**Comment by regginator** [comment]
Completely resolved with release `0.2.2` 🙂 

https://github.com/latte-soft/wax/releases/tag/0.2.2

**Comment by blinxess** [comment]
> Completely resolved with release `0.2.2` 🙂
> 
> https://github.com/latte-soft/wax/releases/tag/0.2.2

That's great, thanks alot. It works very well, good job!!
